controllers/admin/AdminStatusesController.php

```
@@ -629,7 +629,9 @@ public function postProcess()
             if (!$order_state->isRemovable()) {
                 $this->errors[] = $this->trans('For security reasons, you cannot delete default order statuses.', [], 'Admin.Shopparameters.Notification');
             } else {
-                return $order_state->softDelete();
+                $order_state->softDelete();
```

if return false, maybe return an error.